### ISSUE TITLE
undefined method 'without=' for #<Bundler::> (NoMethodError)

### ISSUE THREAD

**Issue by stereobooster** [MOZGIII/bundle-only#2]
I suppose something is changed in new bundler

```
/home/travis/.rvm/gems/ruby-2.3.5/gems/bundle-only-0.2.0/exe/bundle-only:24:in `<top (required)>': undefined method `without=' for #<Bundler::Settings:0x000000000280cb48> (NoMethodError)
	from /home/travis/.rvm/gems/ruby-2.3.5/bin/bundle-only:23:in `load'
	from /home/travis/.rvm/gems/ruby-2.3.5/bin/bundle-only:23:in `<main>'
	from /home/travis/.rvm/gems/ruby-2.3.5/bin/ruby_executable_hooks:15:in `eval'
	from /home/travis/.rvm/gems/ruby-2.3.5/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by MOZGIII** [comment]
I think that's the case. Have no time to fix that right now, so patches are welcome.

**Comment by Startouf** [comment]
Hey @stereobooster did you manage to find some fix ? I just downgraded to bundler 1.14.6 in the meantime...

**Comment by MOZGIII** [comment]
This gem relies on bundler internals. I didn't check it sine the the lasr commit in thia gem. Need to check of that method in bundler code is renamed, on the structure is chaged entirely. I'm not working with ruby projects currently, so I probably won't get to it soon.

**Comment by ciihla** [comment]
Just change to:
```
Bundler.settings.set_command_option :without, groups_to_skip
Bundler.settings.set_command_option :with, install_only
```

**Comment by MOZGIII** [comment]
You mind sending a PR?

**Comment by MOZGIII** [comment]
Closed by #3
